bib/cmd/bootc-image-builder/image.go

```diff
@@ -88,8 +92,6 @@ func manifestForDiskImage(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest
 		"console=ttyS0",
 	}
 
-	img.SysrootReadOnly = true
```

Comment:
Unrelated code changes?

Reply:
Yes, it should be it's own commit, the images library changes so this had to be included so that this even compiles.

Reply:
This is no longer needed because bib is again in sync with the "images" lib API. I removed it :)